tunnel/mock_tunnel.go

```
@@ -89,31 +89,31 @@
 	m.OnSingleBizStatusArrived = OnSingleBizStatusArrived
 }
 
-func (m *MockTunnel) RegisterNode(ctx context.Context, initData model.NodeInfo) {
+func (m *MockTunnel) RegisterNode(initData model.NodeInfo) {
 	return
```

golangci-lint (1.62.2)

S1023: redundant `return` statement

(gosimple)